### PR TITLE
first draft for the new release process

### PR DESCRIPTION
**Pull request by dorin100**
I created a first draft for the new release process. Please review and update it if required.

### CODE REVIEW

**Review by erikd — CHANGES_REQUESTED** [review]
Thsi new version does not mention branches, nor does it mention the `docker-compose.yaml` file, both required for a release.

File: doc/release-process.md [hunk 1]
```diff
@@ -1,24 +1,11 @@
 # Release Process
 
-The rationale for this specific release process is:
+The detailed Release Process is explained [here](https://input-output.atlassian.net/wiki/spaces/REL/pages/1688174696/Release+Process+steps).
```
Comment (rdlrt) [inline doc/release-process.md:3]: Any specific reason to have private links on public repo?

**Comment by dorin100** [comment]
> Thsi new version does not mention branches, nor does it mention the `docker-compose.yaml` file, both required for a release.

@LaurenceIO can you please review/update the DB-Sync Release Process file with your inputs too?

**Comment by erikd** [comment]
I honestly think that rather than simply replacing the old version with a new version, the old version should be fixed and possibly even expanded.

File: doc/release-process.md (hunk 1 above)
Comment (erikd) [reply doc/release-process.md:3]: Yeah, that should not be there.

**Comment by erikd** [comment]
This has not been touched in over 6 months. Closing.

**Pull request closed without merging**